hw-6/controllers/auth.controller.ts

```diff
@@ -0,0 +1,21 @@
+import {Request, Response} from 'express';
+import * as jwt from 'jsonwebtoken';
+import {HTTP_ERROR, JWT_SECRET} from '../constants/http-errors.enum';
+
+function isValidUser(username: string, password: string): boolean {
+  return username === 'admin' && password === 'admin';
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export const login = ({body}: Request, res: Response): void => {
+  const {username, password} = body;
+
+  if (isValidUser(username, password)) {
+    const payload = {user: 'admin'};
```

Comment:
Ideally we should create the payload with information that comes either from the `body` or database?

Reply:
Basically it's up to us what to put there. It must not be sensitive information, since it's base64 encoded (not encrypted). That's why I put only username, just to make sure it works as expected.